orchagent/portsorch.cpp

```diff
@@ -2553,6 +2581,221 @@ bool PortsOrch::removeBridgePort(Port &port)
     return true;
 }
 
+bool PortsOrch::setHostVlanAdminState(int vlan_id, const string &admin_status)
```

Comment:
My understanding is that all the command line settings shall be moved into vlanmgrd instead of in the orchagent code.

Reply:
This is design file
https://github.com/Azure/SONiC/pull/316/files

Reply:
OK, I'll move it all the command line settings into vlanmgrd in next version